tutorials/fundamentals/filter-key.md

```diff
@@ -0,0 +1,36 @@
+To filter an object using the keys, you can use JavaScript's `Object.hasOwnProperty()` method.
+
+```javascript
+const obj = { name: 'Masteringjs', location: 'Florida', help: true };
+let newObj = {}
+if(obj.hasOwnProperty('name')) {
+    newObj['name'] = obj['name'];
+}
+```
+
+## Object.keys()
+
+Another approach to filter using keys is to use the `Object.keys()` method.
```

Comment:
In addition to this `for` loop example, let's add another example using `Object.keys()` and `reduce()`.

```javascript
Object.keys(obj).filter(key => key.includes('name')).reduce((cur, key) => { cur[key] = obj[key]; return cur; }, {});
```